packages/cli/src/commands/join.ts

```diff
@@ -72,15 +64,13 @@ export type JoinOptions = {
   'lint-config'?: RuleSeverity;
 };
 
-export async function handleJoin(argv: JoinOptions, config: Config, packageVersion: string) {
+export async function handleJoin(argv: JoinOptions, config: Config) {
```

Comment:
I believe the `packageVersion` is still needed as it should be used [here](https://github.com/Redocly/redocly-cli/pull/1507/files#diff-3d0894febbd7923f278d468139041923d786a33f480741eabb8d465b9040eb4cR135) instead of the document version (most likely that's a mistake).

Reply:
updated, please take a look